tests/zfs-tests/cmd/devname2devid/Makefile.am

```diff
@@ -5,5 +5,5 @@ pkgexecdir = $(datadir)/@PACKAGE@/zfs-tests/bin
 if WANT_DEVNAME2DEVID
 pkgexec_PROGRAMS = devname2devid
 devname2devid_SOURCES = devname2devid.c
-devname2devid_LDADD = -ludev
+devname2devid_LDADD = $(LIBUDEV_LIBS)
```

Comment:
You should also add LIBUDEV_CFLAGS to devname2devid_CFLAGS I think? It probably works without it because we always add /usr/local/include on FreeBSD.

Reply:
i.e.
`devname2devid_CFLAGS = $(AM_CFLAGS) $(LIBUDEV_CFLAGS)`